packages/fast-components-react-msft/src/action-trigger/action-trigger.props.ts

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import { Omit } from "utility-types";
+import {
+    ActionTriggerClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-msft";
+import {
+    ButtonAppearance,
+    ButtonHandledProps,
+    ButtonManagedClasses,
+    ButtonUnhandledProps,
+} from "../button/button.props";
+
+export enum ActionTriggerAppearance {
+    justified = ButtonAppearance.justified,
+    lightweight = ButtonAppearance.lightweight,
+    outline = ButtonAppearance.outline,
+    primary = ButtonAppearance.primary,
+}
+
+export interface ActionTriggerManagedClasses
+    extends ManagedClasses<ActionTriggerClassNameContract> {}
+export interface ActionTriggerHandledProps
+    extends Omit<ButtonHandledProps, "appearance" | keyof ButtonManagedClasses>,
+        ActionTriggerManagedClasses {
+    /**
+     * The action trigger appearance
+     */
+    appearance?: ActionTriggerAppearance;
+
+    /**
+     * The action trigger glyph render prop
+     */
+    glyph?: (className: string) => React.ReactNode;
```

Comment:
@scomea weren't we going to require glyph as this is part of the design pattern?

Reply:
Yes. Could have sworn I'd changed that already.